src/components/SearchMap.js

```diff
@@ -29,16 +35,19 @@ const SearchMap = ({mapLoaded, lastKnownRegion, updateRegion, sites, navigate})
     const renderSites = () => {
         const renderedSites = _.map(sites, site => {
             const {title, description, coordinate, id} = site;
+            const isSelectedSite = !!selectedSite && id === selectedSite.id;
 
             return (
                 <Marker
+                    onPress={() => getSiteDetail({selectedSite: site})}
```

Comment:
If we click on a marker, set it as the `selectedSite`, but we don't pass `navigate` since we don't want to redirect anywhere.